builders/tools/gsoap.py

```diff
@@ -13,8 +14,12 @@ def install(dir_name,version=None):
         name = 'gsoap_'+str(version)+'.zip'
         try:
             tmp_dir = tempfile.mkdtemp()
+            v = LooseVersion(version)
+            version_short = ".".join(map(str, v.version[:2]))
             path = os.path.join(tmp_dir,name)
-            url = os.path.join('http://downloads.sourceforge.net/project/gsoap2/gSOAP',name)
+            url = os.path.join('http://downloads.sourceforge.net/project/gsoap2',
+                               'gsoap-%s' % version_short,
+                               name)

```

Comment:
I'm tempted to just do this one as 
```python
version_short = '.'.join(version.split('.')[:2])
```
If you don't actually care about the numeric representation for comparison purposes, and just need to match strings, this is probably "good enough".